schema.sql

```diff
@@ -8,3 +8,38 @@ CREATE TABLE animals (
     neutered bool NOT NULL,
     weight_kg decimal NOT NULL
 );
+
+ALTER TABLE animals ADD COLUMN species varchar;
+
+CREATE TABLE owners (
+  id serial PRIMARY KEY,
+  full_name varchar,
+  age int
+);
+
+CREATE TABLE species (
+  id serial PRIMARY KEY,
+  name varchar
+);
+
+ALTER TABLE animals DROP COLUMN species;
+ALTER TABLE animals ADD species_id int REFERENCES species(id);
+ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
+
+CREATE TABLE vets(
+    id SERIAL PRIMARY KEY,
+    name varchar,
```

Comment:
- [ ] Good job here. Kindly add a limit to the name field, which is a good practice in creating a database string field

## sample
- `name varchar(255)`